ksp_plugin/planetarium.hpp

```diff
@@ -88,6 +89,13 @@ class Planetarium {
       Instant const& now,
       bool reverse) const;
 
+  RP2Lines<Length, Camera> PlotMethod2(
```

Comment:
Comment both `PlotMethod1` methods (dunno why the existing one has no comment).

Reply:
I think you mean 2.